auth/services/util.go

```diff
@@ -0,0 +1,82 @@
+package services
+
+import (
+	"crypto"
+	"fmt"
+	"github.com/nuts-foundation/go-did"
+	"github.com/nuts-foundation/nuts-node/vdr/types"
+	"time"
+)
+
+// ResolveSigningKeyID looks up a signing key of the specified holder. It returns the ID
+// of the found key. Typically used to find a key for signing one's own documents. If no suitable keys
+// are found an error is returned.
+func ResolveSigningKeyID(resolver types.DocResolver, holder did.DID, validAt *time.Time) (string, error) {
```

Comment:
What is the test coverage for this file?

Reply:
In crypto, but just used by a unit test... :-p

Reply:
Will remove it